projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -971,7 +969,7 @@ public function get_filtered_connection_data( $selected_post_id = null ) {
 				 * If this is a global connection and this user doesn't have enough permissions to modify
 				 * those connections, don't let them change it.
 				 */
-				if ( ! $done && $this->is_global_connection( $connection_meta ) && ! current_user_can( $this->GLOBAL_CAP ) ) {
+				if ( ! $done && $connection['shared'] && ! current_user_can( $this->GLOBAL_CAP ) ) {
```

Comment:
Should we have a `is_shared` method on the connection or something, so we remove the chance of typos etc? We're also mixing terms 'shared' and 'global'. It would be good to make sure we're only using one of those.

Reply:
Yeah, we can add a method to the connections class to check if a connection is shared.